### PR TITLE
Initial commit to add Postmark to one-click services.

### PR DESCRIPTION
**Pull request by garrettdimon**
This includes entries for DKIM, custom return-path, inbound processing, and DMARC.

### CODE REVIEW

**Comment by garrettdimon** [comment]
Just following up on this to make sure I did everything right since it's been a couple of weeks. No worries if it's just in the queue, but wanted to make sure it didn't slip through the cracks.

**Comment by OleMchls** [comment]
@garrettdimon sorry for letting this sit for two weeks! I'm looking into this this morning :)

File: services/postmark/config.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    "description": "Postmark is a transactional email service to enable your app to reliably send and receive email.",
+    "category": "email",
+    "subdomain-required": false
+  },
+  "fields": [
+    {
+      "name": "dkim_hostname",
+      "label": "DKIM Hostname",
+      "description": "The value from the DKIM 'Hostname' column in your DNS Settings for this domain.",
+      "append": "",
```
Comment (OleMchls) [inline services/postmark/config.json]: Please just omit this key when you do not need to append a domain name for a field.

**Comment by garrettdimon** [comment]
Thanks, Ole! The appends are gone. Let me know if there's anything else I can do.

**Comment by OleMchls** [comment]
Thanks a lot @garrettdimon I will take care of shipping this within the next hours 👍

**Pull request merged**